docs/statistics.md

```diff
@@ -1,267 +1,348 @@
-SRT provides a powerful set of statistical data on a socket.
-This data can be used to keep an eye on a socket's health,
-and track faulty behavior.
+# SRT Statistics
 
-Statistics are calculated independently on each side (receiver and sender),
-and are not exchanged between peers, unless explicitly stated.
+SRT provides a powerful set of statistical data on a socket. This data can be used to keep an eye on a socket's health and track faulty behavior.
 
-The following API functions can be used to retrieve statistics on an SRT socket.
-Refer to the documentation of the [API functions](API-functions.md) for usage instructions.
+Statistics are calculated independently on each side (receiver and sender) and are not exchanged between peers unless explicitly stated.
+
+The following API functions can be used to retrieve statistics on an SRT socket:
 
 * `int srt_bstats(SRTSOCKET u, SRT_TRACEBSTATS * perf, int clear)`
 * `int srt_bistats(SRTSOCKET u, SRT_TRACEBSTATS * perf, int clear, int instantaneous)`
 
-# Total accumulated measurements
+Refer to the documentation of the [API functions](API-functions.md) for usage instructions.
 
-## msTimeStamp
+# Summary Table
+
+The table below provides the summary on the SRT statistics: name, type, unit of measurement, the side it's calculated (sender or receiver), and data type. See section ["Detailed Description"](#Detailed Description) for detailed description of each statistic.
+
+There are three types of statistics:
+
+- Accumulated that means the statistic is accumulated since the time an SRT socket has been created (after the successful call to `srt_connect(...)` or `srt_bind(...)` function), e.g., `pktSentTotal`, etc.,
```

Comment:
```suggestion
- **Accumulated**: the statistic is accumulated since the time an SRT socket has been created (after the successful call to `srt_connect(...)` or `srt_bind(...)` function), e.g., `pktSentTotal`, etc.,
```